airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -1,5 +1,4 @@
-# -*- coding: utf-8 -*- # pylint: disable=too-many-lines
-#
+# -*- coding: utf-8 -*-
```

Comment:
While we're at it, I think we should be consistent with this one as well:
```
# -*- coding: utf-8 -*-
# Licensed to the Apache Software Foundation (ASF) under one
```
or:
```
# -*- coding: utf-8 -*-
#
# Licensed to the Apache Software Foundation (ASF) under one
```
I prefer the latter :-)

Reply:
Right. Fixed :).